### ISSUE TITLE
Crash after exiting Serenitea Pot back to the overworld?

### ISSUE THREAD

**Issue by p0358** [an-anime-team/an-anime-game-launcher#452]
It's not that much of a big deal I'd say, but I'm mostly just curious if anyone else is experiencing a frequent crash of the game when trying to go back to the overworld from Serenitea Pot, using the map. I'm on Arch and AMD APU, using default Wine-Staging-TkG 9.21. It doesn't seem to *always* happen, especially seems not reproducible if I try to test it out with a freshly launched game. But run it for a few hours, go to the teapot, sit in there for like 10-15 minutes at least, then try to go back to overworld and it crashes?

**Comment by Eskuero** [comment]
I also have recently started getting crashes and hangs after teleporting, leaving the pot, into another region, joining a co-op player...

**Comment by p0358** [comment]
Regular teleporting too? Because going into teapot/back, into coop and leaving, has more work going on than a regular teleport, since it crosses the coop barrier (entering or leaving coop; in pot you're technically in a single-person coop world). For me so far I didn't seem to crash from regular teleporting around in the overworld...

**Comment by Eskuero** [comment]
Yeah I got plenty of hangs, mostly teleporting into Inazuma to fish some spots after doing dailies in liyue

**Comment by Eskuero** [comment]
Since the 5.3 update I have not gotten a single crash

**Comment by p0358** [comment]
Indeed I don't think I got a single such crash after the comment above was written, so that's a close then
